dapp/src/queries/useTransactionHistoryQuery.js

```diff
@@ -4,12 +4,12 @@ import { QUERY_KEYS } from '../constants/queryKeys'
 
 import { transactionHistoryService } from '../services/transaction-history.service'
 
-const useTransactionHistoryQuery = (account, options) => {
+const useTransactionHistoryQuery = (account, transactionItems, options) => {
   return useQuery(
     QUERY_KEYS.TransactionHistory(account),
-    () => transactionHistoryService.fetchHistory(account),
+    () => transactionHistoryService.fetchHistory(account, transactionItems),
```

Comment:
This function should accept items per page as second parameter and not total amount of items I think?